### PR TITLE
Catch 404 pages of path: /userland/*

### PR DESCRIPTION
**Pull request by pierreneter**

### CODE REVIEW

File: routes/userland/show.js [hunk 1]
```diff
@@ -1,33 +1,40 @@
...
+      items: report.collection,
+      blacklist: blacklist,
+      page: {
+        title: `${report.title} | Electron`
+      }
+    })
 
-  res.render('userland/show', context)
+    res.render('userland/show', context)
+  } catch (e) {
```
Comment (zeke) [inline routes/userland/show.js]: Rather than catching this entire block of code, can we just detect a 404 earlier and then `return next()`?

File: package.json [hunk 2]
```diff
@@ -19,7 +19,7 @@
     "build-awesome": "script/awesome",
     "build-userland": "script/userland",
     "release": "script/release 2>&1 | script/notify",
-    "dev": "nodemon -e js,scss,yml server.js",
+    "dev": "NODE_PATH=. nodemon -e js,scss,yml server.js",
```
Comment (zeke) [inline package.json]: What is this `NODE_PATH` thing for?

File: server.js [hunk 3]
```diff
@@ -7,11 +7,11 @@ const lobars = require('lobars')
 // Middleware
 const hbs = require('express-hbs')
 const slashes = require('connect-slashes')
-const browsersync = require('./middleware/browsersync')()
+const browsersync = require('./middleware/browsersync')
```
Comment (zeke) [inline server.js]: looks like some unrelated changes ended up in this file?

Comment (pierreneter) [reply server.js]: Yep. I just edit this for consistency in the `require`s. Remove `()` at this line and add to line 55.

File: package.json (hunk 2 above)
Comment (pierreneter) [reply package.json]: Add `NODE_PATH` we can quickly `require` anything in project with a `path` no need to think about  not absolute path. Like I can do here: [routes/userland/show.js#line6](https://github.com/electron/electron.atom.io/pull/799/files/3b56398c6cb4e84666a4c8e1da429ee44b8efe6b#diff-6606e56f8941159c4982e8790c9bcef0R6)
